### ISSUE TITLE
LLaMA LoRA INT4 finetuning issue

### ISSUE THREAD

**Issue by shreyansh26** [stochasticai/xturing#162]
While finetuning the INT4 model, I saw that the loss never decreases, it is always 10.40, irrespective of the data being used. I used both `alpaca_data` and my own custom dataset, it is always the same. I think it is related to #161. 

```
Epoch 0:   1%|          | 27/2500 [01:12<1:51:25,  2.70s/it, v_num=0, loss=10.40]
Epoch 0:   1%|          | 27/2500 [01:12<1:51:25,  2.70s/it, v_num=0, loss=10.40]
Epoch 0:   1%|          | 28/2500 [01:15<1:50:37,  2.69s/it, v_num=0, loss=10.40]
Epoch 0:   1%|          | 28/2500 [01:15<1:50:37,  2.69s/it, v_num=0, loss=10.40]
Epoch 0:   1%|          | 29/2500 [01:17<1:49:53,  2.67s/it, v_num=0, loss=10.40]
Epoch 0:   1%|          | 29/2500 [01:17<1:49:53,  2.67s/it, v_num=0, loss=10.40]
Epoch 0:   1%|          | 30/2500 [01:19<1:49:13,  2.65s/it, v_num=0, loss=10.40]
Epoch 0:   1%|          | 30/2500 [01:19<1:49:13,  2.65s/it, v_num=0, loss=10.40]
Epoch 0:   1%|          | 31/2500 [01:21<1:48:34,  2.64s/it, v_num=0, loss=10.40]
Epoch 0:   1%|          | 31/2500 [01:21<1:48:34,  2.64s/it, v_num=0, loss=10.40]
Epoch 0:   1%|▏         | 32/2500 [01:24<1:47:58,  2.63s/it, v_num=0, loss=10.40]
Epoch 0:   1%|▏         | 32/2500 [01:24<1:47:58,  2.63s/it, v_num=0, loss=10.40]
Epoch 0:   1%|▏         | 33/2500 [01:26<1:47:24,  2.61s/it, v_num=0, loss=10.40]
Epoch 0:   1%|▏         | 33/2500 [01:26<1:47:24,  2.61s/it, v_num=0, loss=10.40]
Epoch 0:   1%|▏         | 34/2500 [01:28<1:46:54,  2.60s/it, v_num=0, loss=10.40]
Epoch 0:   1%|▏         | 34/2500 [01:28<1:46:54,  2.60s/it, v_num=0, loss=10.40]
Epoch 0:   1%|▏         | 35/2500 [01:30<1:46:25,  2.59s/it, v_num=0, loss=10.40]
```
Architecture Details - using a T4 GPU.

**Comment by StochasticRomanAgeev** [comment]
Hi @brijeshKarna,
It looks like `pytorch_lightning` issue, we will work on fixing this in latest update.
But you also can create pr to make logging working!
We would appreciate any help provided in this case
